app/views/manuals/updates.html.erb

```diff
@@ -1,3 +1,9 @@
+<% content_for :extra_headers do %>
```

Comment:
After consultation with Alan Madrell, I've gone for "List of updates to '[name of manual].'